app/views/complaints/_tta_activity_report_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="ct-section-divider">
+  <p class="display-flex flex-justify">
+    <span> Activity Report ID</span>
+    <span><%= link_to "Cancel", "#js-add-tta-report", id: "js-close-tta-form", role: "button" %></span>
+    <div>
+      <%= form_with url: issue_tta_reports_path, method: "post", class: "usa-search display-flex", id: "tta-report-form"  do %>
+        <%= label_tag "tta-report-id", "TTA report display id", id: "tta-report-label" %>
+        <%= text_field_tag :tta_report_id, nil, id: "tta-report-id", "aria-describedBy": "tta-report-label" %>
```

Comment:
[thought] `aria-describedBy` works well. You can also accomplish the same thing by removing the CSS from the `label#tta-report-label` and adding the `usa-sr-only` class to it. I think `visibility: hidden` causes the label to be hidden from VO as well as from sighted users

Reply:
i did all my experimenting with the label before i hid it. when it was visible it was not triggering in the screenreader until i used `aria-describedBy`

Reply:
i do prefer the `usa-sr-only` class to creating my own custom css, though, so i am going to do that